scss/utils/vars/_type.scss

```diff
@@ -14,6 +14,8 @@ $font : "Graphik Meetup", helvetica, arial, sans-serif;
 $font--mono : Monaco, "Andale Mono", "Courier New", monospace;
 
 /// @type Value(px)
+$font-size-tiny      : 12px;
```

Comment:
nitpick - do we have `tiny` used elsewhere , or always `xsmall`

Reply:
We usually use `xsmall`.  I named it `tiny` to emphasize how unusual it will be to use this size.

If that's confusing (or just too opinionated), I'll gladly change to `xsmall` :)

Reply:
its not confusing... I get it